mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -595,14 +595,35 @@ class Completions(
         Some(search.search(query, buildTargetIdentifier, visitor))
       case CompletionKind.Members =>
         val visitor = new CompilerSearchVisitor(sym =>
-          if sym.is(ExtensionMethod) &&
+          def isExtensionMethod = sym.is(ExtensionMethod) &&
             qualType.widenDealias <:< sym.extensionParam.info.widenDealias
-          then
+
+          def isImplicitClass(owner: Symbol) =
+            val constructorParam =
+              owner.info.allMembers
+                .find(_.symbol.isAllOf(Flags.PrivateParamAccessor))
```

Comment:
```suggestion
                .find(_.symbol.isAllOf(Flags.ParamAccessor))
```
It seems it doesn't have to be private and 
```Scala
implicit class A (val num: Int):
  def incr: Int = num + 1 
```
is legal.

Reply:
You're right, fixed and changed one of the tests.